docs/howto_firewall.md

```diff
@@ -17,11 +19,12 @@ The following ports are optional and they should be considered for MicroShift if
 |443        |TCP        |HTTPS port used to serve applications through the OpenShift router |
 |5353       |UDP        |mDNS service to respond for OpenShift route mDNS hosts |
 |30000-32767|TCP/UDP    |Port range reserved for NodePort type of services, can be used to expose applications on the LAN |
-|6443       |TCP        |HTTPS API port for the MicroShift API |
+|6443       |TCP        |HTTPS port for the MicroShift API |
 
 ## Firewalld
 The following commands can be used for enabling `firewalld` and opening all the above mentioned source IP addresses and ports.
 > Use the appropriate pod IP range if it is different from the default `10.42.0.0/16` setting.
+> Use the appropriate optional settings when requiring external access to services running in MicroShift.
```

Comment:
I'm not sure if this sentence can be understood without some context/examples.

Reply:
Added some examples.